sklearn_numba_dpex/kmeans/kernels/compute_labels.py

```diff
@@ -35,6 +35,7 @@ def make_label_assignment_fixed_window_kernel(
     ) = _make_initialize_window_kernel_funcs(
         n_clusters,
         n_features,
+        True,
```

Comment:
Maybe move this boolean flag as the last argument to make it possible to pass it as a named argument to improve readability.